Adding Fine-tuning to Get-Started section (#113)
kluster-ai/docs: get-started/fine-tuning/overview.md

---

> title: Overview of Fine-tuning models with kluster.ai

Reviewer: ```suggestion
title: Overview of Fine-tuning models
```

Fine-tuning is particularly valuable when you want consistent, specialized responses that would otherwise require complex prompting or when you need the model to reliably generate outputs in a specific format.

## When to fine-tune your model

Fine-tuning is ideal for scenarios where you need:

- **Domain specialization**: Create models that excel in specific fields like medicine, law, finance, or technical documentation.
- **Brand-aligned responses**: Train models to match your company's voice, style, and communication guidelines.
- **Format consistency**: Ensure reliable output in specific formats like JSON, XML, or Markdown.
- **Enhanced reasoning**: Improve analytical capabilities for specific types of problems.
- **Custom behavior**: Develop assistants that follow your unique processes and workflows.

## Benefits of fine-tuning

Fine-tuning delivers several key advantages over using general-purpose models:

- **Improved performance**: Fine-tuned models consistently outperform base models on specific tasks.
- **Cost efficiency**: Smaller fine-tuned models can match or exceed the performance of larger models at a lower cost.

> - **Reduced latency**: Fine-tuned models deliver faster responses, improving user experience.

Reviewer: ```suggestion
- **Reduced latency**: Fine-tuned models provide faster responses, enhancing the user experience.
```

- **Consistency**: Achieve more reliable outputs tailored to your specific requirements.
- **Data privacy**: Train models on your data without exposing sensitive information in prompts.

## Supported models

> kluster.ai currently supports fine-tuning for two powerful foundation models:

Reviewer 2: ```suggestion
kluster.ai currently supports fine-tuning for the following models:
```

--8<-- 'text/get-started/fine-tuning-supported-models.md'

## Choose your fine-tuning approach

kluster.ai offers two ways to fine-tune models, each designed for different user preferences and requirements:

<div class="grid cards" markdown>

-   <span class="badge guide">Guide</span> __Platform__

    ---

    Use the platform to fine-tune, without writing code. The platform is ideal for users who want a guided, interactive experience and real-time feedback on training progress.

    [:octicons-arrow-right-24: Visit the guide](/get-started/fine-tuning/platform/){target=_blank}

-   <span class="badge guide">Guide</span> __API__

    ---

    Fine-tune models with code for maximum flexibility and automation. The API is best for developers who need advanced customization, integration, or workflow automation.

    [:octicons-arrow-right-24: Visit the guide](/get-started/fine-tuning/api/){target=_blank}

</div>

## Additional resources

> - **Step-by-step tutorial**: Learn the fundamentals with our [Fine-tuning Sentiment Analysis Tutorial](https://docs.kluster.ai/tutorials/klusterai-api/finetuning-sent-analysis/){target=_blank}.

Reviewer: ```suggestion
- **Step-by-step tutorial**: Learn the fundamentals with our [Fine-tuning sentiment analysis tutorial](/tutorials/klusterai-api/finetuning-sent-analysis/){target=_blank}.
```
